assignments/arrays.js

```diff
@@ -75,30 +75,69 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+
+function idSearch (idNum){
+  for (let i in inventory) {
+    if (inventory[i].id === idNum){
+      return "Car " + idNum + " is a " + inventory[i].car_year + ", " + inventory[i].car_make + ", " + inventory[i].car_model + ".";
+    }
+  }
+};
+
+console.log(idSearch(33));
```

Comment:
Note that `for...in` loop is preferrable for looping through the properties of an object, though it works for arrays, it's not a good fit if the order of the index is important. Reference https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in